web/src/main/java/com/futurewei/alcor/web/entity/NetworkVxlanRange.java

```diff
@@ -0,0 +1,75 @@
+package com.futurewei.alcor.web.entity;
+
+import com.futurewei.alcor.common.exception.KeyInvalidException;
+import com.futurewei.alcor.common.exception.NetworkKeyAllocNotFoundException;
+import com.futurewei.alcor.web.allocator.NetworkKeyAllocator;
+import lombok.Data;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Data
+public class NetworkVxlanRange {
+
+    private String id;
+    private String segmentId;
```

Comment:
I think segmentId should be removed from NetworkVxlanRange as well.

Reply:
Yeah